cloud/high-availability.md

```diff
@@ -1,46 +1,126 @@
 # High availability
-Timescale Cloud uses various methods to ensure that your service remains
-available. This section covers some of the strategies used to
-increase service availability.
+Availability needs are determined by how a system recovers when a 
+database crashes or becomes unavailable. Availability is a balance 
+between how much data may be lost after a failure, the Recovery Point 
+Objective (RPO), and how quickly the database is back up and running, 
+the Recovery Time Objective (RTO). High availability usually refers to 
+keeping the system operational during a failure while minimizing any 
+data loss. 
 
-High availability (HA) is achieved by increasing redundancy and
-resilience. To increase redundancy, parts of the system are
-replicated, so that they are on standby in the event of a failure. To increase
-resilience, recovery processes switch between these standby resources as quickly
-as possible.
+Timescale Cloud provides very low RTO and RPO for all instances 
+(see rapid recovery below). Timescale Cloud also offers features like 
+replicas to satisfy high availability requirements to ensure near-zero 
+downtime and data loss in the event of a failure. This section covers 
+some strategies used to increase service availability on Timescale 
+Cloud.
 
-## Backups
-On Timescale Cloud, full backups are taken weekly and incremental backups are
-performed daily.
+## Rapid recovery
+For all services, Timescale Cloud decouples the database's compute and 
+storage, allowing Cloud to provide ways for databases to self-heal 
+gracefully rather than crashing and recovering from backup in all 
+failure scenarios. Historically, a failure in any part of the database 
```

Comment:
```suggestion
For all services, Timescale Cloud decouples the database's compute and
storage. This allows databases to self-heal gracefully in many situations,
rather than crashing and recovering from backup in all cases.
```